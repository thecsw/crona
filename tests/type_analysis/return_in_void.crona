main : void() {
    return 3+true;
}