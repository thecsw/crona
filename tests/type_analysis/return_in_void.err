FATAL [2,13]: Return with a value in void function
Type Analysis Failed
