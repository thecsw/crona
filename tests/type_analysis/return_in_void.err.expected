FATAL [2,14]: Arithmetic operator applied to invalid operand
FATAL [2,13]: Return with a value in void function
Type Analysis Failed